cas/audit-incident-report.md

```diff
@@ -0,0 +1,88 @@
+# Audit Incident Reporting Guidelines
+
+## Change History
+
+|Version|Effective Date|
+|-|-|
+|2.1 (current)|TBD| 
+|[2.0](https://github.com/mozilla/www.ccadb.org/blob/master/incident_archive/ir_version_2_0.md)|October 17, 2023| 
+|[1.0](https://github.com/mozilla/www.ccadb.org/blob/master/incident_archive/ir_version_1_0.md)|February 15, 2023|
+
+## Audit Incident Reporting
```

Comment:
Shouldn't it be "Audit Finding Reporting"?
A finding does not have to be an incident... (applies also for the rest of the template)

Reply:
@romanf I believe the intent of using the naming "Audit Incident Report" is to differentiate that type of report and its contents from a "Incident Report". Depending on Root Program Policies the "Audit Incident Report" may be required for any non-conformities, qualifications, or modified opinions included in an Audit. The "Audit Incident Report" may include one or more findings, but it is focused on the root cause of each and action items to address the root cause. The other type of report (i.e., "Incident Report") intends to include even more content.

Reply:
@ChristopherRC I was probably not clear enough. I think the term "Audit Incident Report" should be "Audit Report". Often the audit report does not contain any findings (and even if, a finding is not an incident).

Reply:
#187 now describes these as "Audit Finding Reports."